rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -316,7 +316,7 @@ unsigned rocalTensor::copy_data(hipStream_t stream, void *host_memory, bool sync
 #endif
 
 unsigned rocalTensor::copy_data(void *user_buffer) {
-    if (_info._type != rocalTensorInfo::Type::HANDLE) return 0;
```

Comment:
why is this changed? Are they both have the same meaning

Reply:
In our previous case we would associate all the image related rocalTensors with a vx_tensor and we set the type as handle for the output tensors where we create vx_tensors with handle.
But with our current changes we are not creating the output tensor (which is to be returned to the user) as a vx_tensor hence handle is not set, so we could not copy the data. Hence I have changed the condition to check if it is a null pointer.